src/css/themes/po-theme-default.css

```
@@ -1570,9 +1570,18 @@ po-textarea {
   /*------------------------------------*\
     TOOLTIP
   \*------------------------------------*/
+}
 
-  --color-tooltip-background-color-balloon: var(--color-neutral);
-  --color-tooltip-color-text-balloon: var(--color-white);
+.po-tooltip {
```

Seguir os padrões dos outros componentes utilizando apenas o componente e não a classe.

```suggestion
po-tooltip {
```